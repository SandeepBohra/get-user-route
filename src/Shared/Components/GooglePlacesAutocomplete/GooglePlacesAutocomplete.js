import React, { Component } from 'react';
import './GooglePlacesAutocomplete.css'

class GooglePlacesAutocomplete extends Component {
    render() {
        return(
            <div className="GooglePlacesAutocomplete">
                <p>{this.props.label}</p>
                <input 
                    type="text" 
                    name={this.props.inputName} 
                    ref={this.props.fieldRef}
                    onChange={this.props.handleInputChange}
                />
            </div>
        )
    }
}

export default GooglePlacesAutocomplete;