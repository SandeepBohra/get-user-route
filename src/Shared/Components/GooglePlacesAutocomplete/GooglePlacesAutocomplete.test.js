import React from 'react';
import renderer from 'react-test-renderer';
import GooglePlacesAutocomplete from './GooglePlacesAutocomplete';
import { shallow } from 'enzyme';

describe('<GooglePlacesAutocomplete /> component', () => {
  
  it('should match the snapshot', () => {
    const output = renderer.create(<GooglePlacesAutocomplete />).toJSON();
    expect(output).toMatchSnapshot();
  })

  it('render HTML correctly', () => {
    const wrapper = shallow(<GooglePlacesAutocomplete />);
    
    expect(wrapper.find('input').length).toBe(1);
  });
})