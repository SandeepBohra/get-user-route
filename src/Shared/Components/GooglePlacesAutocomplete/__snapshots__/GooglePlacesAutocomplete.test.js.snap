// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GooglePlacesAutocomplete /> component should match the snapshot 1`] = `
<div
  className="GooglePlacesAutocomplete"
>
  <p />
  <input
    type="text"
  />
</div>
`;
